src/Config.js

```diff
@@ -143,6 +143,10 @@ class Config {
       // Can add additional transfer targets, e.g., discovery
     }
   }
+
+  static get defaultLanguageId() {
+    return "eng"
+  }
 }
```

Comment:
as requested by Justin L, moved this value from `Utilities` to `Config` -- and then adjustments to it's use